packages/rollup-plugin-styles/src/loaders/postcss/icss/index.ts

```
@@ -0,0 +1,50 @@
+const plugin: PluginCreator<InteroperableCSSOptions> = (options = {}) => {
+    const load = options.load ?? loadDefault;
+    const extensions = options.extensions ?? extensionsDefault;
+
+    return {
+        async OnceExit(css, { result }) {
+            if (!css.source?.input.file) {
+                return;
+            }
+
+            const resultOptions: Result["opts"] = { ...result.opts };
+            delete resultOptions.map;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)